Cargo.toml

```diff
@@ -21,7 +21,7 @@ core = { version = "1.0", optional = true, package = "rustc-std-workspace-core"
 libc = { version = "0.2.64", default-features = false }
```

Comment:
Can you update this to:
```toml
[target.'any(cfg(unix), target_os = "wasi")'.dependencies]
libc = { ... }
```
We shouldn't need to increase the crate version as https://github.com/rust-lang/libc/commit/58ba8129c4a46b7e61ec61cd61c32d2082fefcd8 was added in `libc` 0.2.53